frontend/lib/norent/letter-builder/routes.ts

```diff
@@ -5,6 +5,11 @@ export type NorentLetterBuilderRouteInfo = ReturnType<
   typeof createNorentLetterBuilderRouteInfo
 >;
 
+/**
+ * This function maps URL paths to our routes within the NoRent Letter Builder flow.
+ * To find the actual definition of these routes, check out the
+ * getNoRentLetterBuilderProgressRoutesProps function in the `steps.tsx` file in the same directory as this file.
```

Comment:
So one thing I'm super paranoid of is "doc rot", where documentation becomes outdated relative to the code it's describing, at which point it could potentially be more confusing than it is helpful... in this case I'm a tiny bit worried about the `getNoRentLetterBuilderProgressRoutesProps` getting renamed at some point without the committer making a corresponding change to this documentation, which would then make it misleading.  I think that `steps.tsx` is small enough that this comment can just refer to it without explicitly referring to the function that defines the routes, so if I were writing this I'd just leave out the name of the function, but it's up to you!  Either way not a big deal.

Reply:
That totally makes sense! Yeah, I was not sure whether to include the function name or not, but you've convinced me— I think it'll be clear enough especially since we have comments on both pages.